src/content/app/tools/blast/views/blast-submission-results/components/blast-results-per-sequence/BlastResultsPerSequence.tsx

```diff
@@ -64,72 +66,97 @@ const BlastResultsPerSequence = (props: BlastResultsPerSequenceProps) => {
     ? styles.headerWithLegend
     : undefined;
 
-  return (
-    <div className={styles.wrapper}>
-      <div className={styles.resultsSummaryRow}>
-        <div>Sequence {sequence.id}</div>
-        <div className={styles.sequenceHeader}>
-          <div className={headerClasses}>
-            <ShowHide
-              label={sequenceHeaderLabel}
-              isExpanded={shouldShowParamaters}
-              onClick={() => showParamaters(!shouldShowParamaters)}
-            ></ShowHide>
-            {shouldUseGenomicHitsDiagram && <BlastGenomicHitsDiagramLegend />}
+  const hasAllJobsFailed = blastResults.every(
+    (job) => job.status === 'FAILURE'
+  );
+
+  if (hasAllJobsFailed) {
```

Comment:
- What happens if there is only one job per sequence?
- I think you are only covering [this XD](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/566d3729-59d6-444a-8774-66d84af759dc?fullscreen), where all jobs per sequence have failed. But there is [another XD](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/88c148e3-c947-4203-b1ca-fcd52901c4a4?fullscreen), where only some jobs per sequence have failed